stock_move_line_quant/views/stock_picking_views.xml

```diff
@@ -5,6 +5,12 @@
         <field name="model">stock.picking</field>
         <field name="inherit_id" ref="stock.view_picking_form" />
         <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='move_line_ids_without_package']//field[@name='lot_id']"
+                position='attributes'
+            >
+                <attribute name="options"> {"create":True,"create_edit":True} </attribute>
```

Comment:
```suggestion
                <attribute name="options">{"create":True,"create_edit":True}</attribute>
```
A minor styling preference. Remove unnecessary spaces.

Reply:
@yostashiro 
Thank you, corresponded it.